login-workflow/CHANGELOG.md

```diff
@@ -5,27 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.1.0 (Unreleased)
```

Comment:
You can probably put the date in here now if you think we can finish it up and make a master PR to publish this today since this is the last piece for 2.1.0. Or if you want to be safe you can use tomorrow's date.